flocker/acceptance/test_api.py

```diff
@@ -382,6 +382,33 @@ def test_linking(self, cluster):
                 origin.public_address, origin_port))
         return running
 
+    @require_cluster(2)
+    def test_traffic_routed(self, cluster):
+        """
+        An application can be accessed even from a connection to a node
+        which it is not running on.
+        """
+        port = 8080
```

Comment:
Oh ignore that. I already knew that `find_free_port` operates locally and won't help find a cluster-wide free port.